scripts/client.js

```diff
@@ -289,6 +282,8 @@ const processFile = async (name) => {
 			// 	Date.now() + finishTime,
 			// ).toLocaleTimeString()}`,
 		);
+		//not adding to the previous mess, incase the finish time is added back or something
+		process.stdout.write("      \r");
```

Comment:
Finish time was removed after I made it resolve multiple promises at once. Not sure how I'll calculate a finish time now

Reply:
Ah, i still think in the context of that one that its better to keep it as a seprate call to write, it doesn't really matter in the end.